### DISCUSSION TITLE
How to change Card size?

### DISCUSSION THREAD

**Discussion by Caresle** [themesberg/flowbite-vue#220] [Q&A] [answered]
I'm trying to change the default size of the card to full width, but i can achive that, i've tried:

- `w-full` in the Card component
- `min-w-full` in the Card component
- `max-w-full` in the Card component
- wrapping the card in a div and add the w-full, min-w-full and max-w-full in the div
- Adding `style="width: 100% !important;"` in the card

i couldn't find a size option in the [docs](https://flowbite-vue.com/components/card)

**Comment by cogor** [comment, marked as answer]
Hey, you can apply `!max-w-full` class to the component.
P.S. I will refactor this component soon and there will be no need to use "!" to override a class
```vue
    <the-card href="#" class="!max-w-full">
      <h5 class="mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">Noteworthy technology acquisitions 2021</h5>
      <p class="font-normal text-gray-700 dark:text-gray-400">Here are the biggest enterprise technology acquisitions of 2021 so far, in reverse chronological order.</p>
    </the-card>
```

**Reply by Caresle** [reply]
Thank you
